cmd/juju/commands/bootstrap.go

```diff
@@ -536,6 +552,11 @@ func (c *bootstrapCommand) Run(ctx *cmd.Context) (resultErr error) {
 		}
 		inheritedControllerAttrs[k] = v
 	}
+	// Model defaults are added to the inherited controller attributes.
+	// Any command line set model defaults override what is in the cloud config.
+	for k, v := range modelDefaultConfigAttrs {
```

Comment:
We should check and error if they try and specify a bootstrap or controller only attribute in the model defaults. Maybe easiest to use a switch{} inside a for{} similar to above
(and a test)